### PR TITLE
`stub-convention` plugin

### PR DESCRIPTION
**Pull request by turansky**
Part of #1938

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -9,3 +11,14 @@ repositories {
 kotlinDslPluginOptions {
     experimentalWarning.set(false)
 }
+
+val props = Properties().apply {
```
Comment (qwwdfsad) [inline buildSrc/build.gradle.kts:15]: please rename it to just `properties`

File: js/js-stub/build.gradle.kts [hunk 2]
```diff
@@ -2,10 +2,6 @@
...
  */
 
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
-tasks.named<KotlinCompile>("compileKotlin") {
-    kotlinOptions {
-        freeCompilerArgs += "-Xallow-kotlin-package"
-    }
+plugins {
+    `stub-convention`
```
Comment (qwwdfsad) [inline js/js-stub/build.gradle.kts:6]: Please rename it to `stdlib-stub-convention`, just "stub" is too broad term

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (turansky) [reply buildSrc/build.gradle.kts:15]: `properties` already exists in `Project`

File: js/js-stub/build.gradle.kts (hunk 2 above)
Comment (turansky) [reply js/js-stub/build.gradle.kts]: Done

**Pull request closed without merging**